File: lee-hankins-backend-capstone/Controllers/CustomerController.cs
using lee_hankins_backend_capstone.Models;
using System.Collections.Generic;
using System.Web.Http;

namespace lee_hankins_backend_capstone.Controllers
{
    public class CustomerController : ApiController
    {
        private CustomerRepository _customerRepository;

        public CustomerController(CustomerRepository customerRepository)
        {
            _customerRepository = customerRepository;
        }

        // GET: api/Customer
        public IEnumerable<Customer> Get()
        {
            return _customerRepository.GetAll();
        }

        // GET: api/Customer/
        public string Get(int id)
        {
            return "value";
        }

        // POST: api/Customer
        public void Post(Customer newCustomer)
        {
            _customerRepository.Save(newCustomer);
        }

        // PUT: api/Customer/5
        public void Put(int id, [FromBody]string value)
        {
        }

        // DELETE: api/Customer/5
        public void Delete(int id)
        {
            _customerRepository.Delete(id); 
        }
    }
}

File: lee-hankins-backend-capstone/app/controllers/customerController.js
app.controller("customerController", ["$scope", "$http", "$location", function ($scope, $http, $location) {

    $scope.customers = [];
    $scope.newCustomer = {
        FirstName: "",
        LastName: "",
        Email: ""
    }


    $http.get("/api/customer")
    .then(function (result) {
        $scope.customers = result.data;
    });
    

    $scope.addCustomer = function () {
        $http.post("/api/customer", $scope.newCustomer)
        .then(function (result) {
            $location.path("new-order");
        });
    }

    $scope.delete = function (customer) {
        $http.delete("api/customer/" + customer.CustomerId);
        $http.get("/api/customer")
            .then(function (result) {
                $scope.customers = result.data;
            });
    }
}])